packages/generate/src/genutil.ts

```diff
@@ -4,6 +4,7 @@ import type {
   DirBuilder,
   IdentRef,
 } from "./builders";
+import { getPackageVersion } from "./commandutil";
```

Comment:
Should I just move the const there instead?

Reply:
It's only used here now, so I'd say just make the string itself a const in this file and use it inline like:

```ts
const PACKAGE_VERSION_TEMPLATE = `__@edgedb/generate__VERSION_PLACEHOLDER__`;

export const headerComment = `// GENERATED by @edgedb/generate v${PACKAGE_VERSION_TEMPLATE}`;
```

I'm not sure why it was a function before (maybe it did the look-up dynamically instead of getting replaced later?)